spec/features/admin/orders/payments_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'payments', type: :feature do
+  stub_authorization!
+
+  let(:order) { create(:order) }
+  let!(:gateway) { create_braintree_payment_method }
+
+  it 'allows you to create a new credit card' do
+    visit spree.admin_order_payments_path(order)
+    expect(page).to have_css("#card_new#{gateway.id}")
```

Comment:
Instead of targeting the CSS selector (which is an implementation detail), this could be `choose("Use a new card")`